packages/configs/default/index.json

```diff
@@ -4,6 +4,16 @@
     "types:*.{ts,tsx}": ["@parcel/transformer-typescript-types"],
     "bundle-text:*": ["@parcel/transformer-inline-string", "..."],
     "data-url:*": ["@parcel/transformer-inline-string", "..."],
+    "worker:*": [
+      "@parcel/transformer-worker",
+      "...",
+      "@parcel/transformer-clear"
+    ],
+    "worklet:*": [
+      "@parcel/transformer-worklet",
```

Comment:
Since the source code for these is identical, maybe it would be more efficient to make 1 transformer and merely have 2 pipelines here point to the same transformer configuration?

Reply:
They aren't identical. Worklets cannot use `importScripts`. That's the TODO part of this branch